templates/workflows/docker_module.yml

```diff
@@ -12,21 +12,34 @@ concurrency:
   group: "{{ openscpca_module }}-${{ github.ref }}"
   cancel-in-progress: true
 
-on:
-  # pull_request:
-  #   branches:
-  #     - main
-  #   paths:
-  #     - "analyses/{{ openscpca_module }}/Dockerfile"
-  #     - "analyses/{{ openscpca_module }}/.dockerignore"
-  #     - "analyses/{{ openscpca_module }}/renv.lock"
-  #     - "analyses/{{ openscpca_module }}/conda-lock.yml"
-  workflow_dispatch:
-    inputs:
-      push-ecr:
-        description: "Push to AWS ECR"
-        type: boolean
-        required: false
+# on:
+#   pull_request:
+#     branches:
+#       - main
+#     paths:
+#       - analyses/simulate-sce/Dockerfile
+#       - analyses/simulate-sce/.dockerignore
+#       - analyses/simulate-sce/renv.lock
+#       - analyses/simulate-sce/conda-lock.yml
```

Comment:
I kind of preferred `{{ openscpca_module }}` here. Convince me otherwise?

Reply:
This is an error.